src/openfermion/hamiltonians/_jellium.py

```diff
@@ -405,12 +291,14 @@ def jordan_wigner_dual_basis_jellium(grid, spinless=False,
         grid (Grid): The discretization to use.
         spinless (bool): Whether to use the spinless model or not.
         include_constant (bool): Whether to include the Madelung constant.
+            Note: constant is unsupported for non-uniform, non-cubic cells with
```

Comment:
I think that this indentation and writing "Note:" is going to confuse the automatic docs. Because it is likely going to think now that "Note" is an argument that is supposed to be passed to this function. Perhaps you can just remove the colon and write on this line "note that the constant is not..."